File: ShoeStore/ShoeStore.Domain/Entities/Brand.cs
namespace ShoeStore.Domain.Entities;

public class Brand
{
    public int Id { get; set; }
    public string? Code { get; set; }
    public string Name { get; set; } = null!;
    public string? Description { get; set; }
    public int StatusId { get; set; }

    public Status Status { get; set; } = null!;
    public ICollection<Product>? Products { get; set; }
}

File: shoestorefe/lib/domain/entities/brand.dart
class Brand {
  final int id;
  final String? code;
  final String name;
  final String? description;
  final int statusId;

  Brand({
    required this.id,
    this.code,
    required this.name,
    this.description,
    required this.statusId,
  });
}
